pkg/multicluster/target.go

```diff
@@ -36,7 +36,7 @@ func (t *GatewayTarget) GetName() string {
 }
 
 func (t *GatewayTarget) GetShortCode() string {
-	return ToBase36hash(t.GetName())
+	return ToBase36HashLen(t.GetName(), 7)
```

Comment:
Why a length 7? Was 6 not long enough?

Reply:
ah, this was unintended, I'll set it back.